simple pr template (#5986)
desktop/desktop: .github/pull_request_template.md

## Overview

> <!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->

Reviewer: ```suggestion
<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose before opening this PR.) -->
```

> <!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->

Reviewer: ```suggestion
<!-- Please insert the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->
```

> <!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->

Reviewer: ```suggestion
<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues to allow for discussion before opening this PR.) -->
```

> <!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->

Reviewer 2: ```suggestion
<!-- 
What issue are you addressing? e.g. #1234 
If there is no issue for this pull request (PR), please create one before opening your PR.
Issues can be created at https://github.com/desktop/desktop/issues) 
-->
```

I think ☝️ 
* Increases readability by separating content from markup
* Question makes clear what we're looking for
* Helps guide user by giving an example
* Defines acronym before using it

> <!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose before opening this PR.) -->

Reviewer: ```suggestion
<!-- Please insert the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose to allow for discussion before opening this PR.) -->
```

> **resolves #[issue number]**

Reviewer 2: Let's use simpler words.

```suggestion
**Fixes #[issue number]**
```

> -

Reviewer: Does the bullet here have a purpose? I might consider leaving that out?

## Overview

<!--
What issue are you addressing? (for example, #1234)

If an issue doesn't exist for this pull request (PR) to address, please open one to allow for discussion before opening this PR.
(You can open a new issue at https://github.com/desktop/desktop/issues/new/choose)
-->

> **Closes #{issue number}**

Reviewer 2: Will this formatting prevent GitHub from automagically linking this issue?
